lib/internment.rs

```diff
@@ -138,66 +126,7 @@ where
     }
 }
 
-#[cfg(feature = "flatbuf")]
-impl<A, FB> FromFlatBuffer<FB> for Intern<A>
-where
-    A: Eq + Hash + Send + Sync + 'static,
-    A: FromFlatBuffer<FB>,
-{
-    fn from_flatbuf(fb: FB) -> Result<Self, String> {
-        Ok(Intern::new(A::from_flatbuf(fb)?))
-    }
-}
-
-#[cfg(feature = "flatbuf")]
-impl<'b, A, T> ToFlatBuffer<'b> for Intern<A>
-where
-    T: 'b,
-    A: Eq + Send + Sync + Hash + ToFlatBuffer<'b, Target = T>,
-{
-    type Target = T;
-
-    fn to_flatbuf(&self, fbb: &mut fbrt::FlatBufferBuilder<'b>) -> Self::Target {
-        self.as_ref().to_flatbuf(fbb)
-    }
-}
-
-/*#[cfg(feature = "flatbuf")]
-impl<'a> FromFlatBuffer<fb::__String<'a>> for intern_istring {
-    fn from_flatbuf(v: fb::__String<'a>) -> Response<Self> {
-        Ok(intern_string_intern(&String::from_flatbuf(v)?))
-    }
-}*/
-
-#[cfg(feature = "flatbuf")]
-impl<'b, A, T> ToFlatBufferTable<'b> for Intern<A>
-where
-    T: 'b,
-    A: Eq + Send + Sync + Hash + ToFlatBufferTable<'b, Target = T>,
-{
-    type Target = T;
-    fn to_flatbuf_table(
-        &self,
-        fbb: &mut fbrt::FlatBufferBuilder<'b>,
-    ) -> fbrt::WIPOffset<Self::Target> {
-        self.as_ref().to_flatbuf_table(fbb)
-    }
-}
-
-#[cfg(feature = "flatbuf")]
-impl<'b, A, T> ToFlatBufferVectorElement<'b> for Intern<A>
-where
-    T: 'b + fbrt::Push + Copy,
-    A: Eq + Send + Sync + Hash + ToFlatBufferVectorElement<'b, Target = T>,
-{
-    type Target = T;
-
-    fn to_flatbuf_vector_element(&self, fbb: &mut fbrt::FlatBufferBuilder<'b>) -> Self::Target {
-        self.as_ref().to_flatbuf_vector_element(fbb)
-    }
-}
-
-pub fn istring_join(strings: &crate::ddlog_std::Vec<istring>, sep: &String) -> String {
+pub fn istring_join(strings: &ddlog_std::Vec<istring>, sep: &String) -> String {
```

Comment:
```suggestion
pub fn istring_join(strings: &[istring], sep: &str) -> String {
```
Deref coercion should allow `&String` -> `&str` and `&ddlog_std::Vec<T>` -> `&[T]`, may need to implement `Deref<Target = Vec<T>>` and `Deref<Target = [T]>` for `ddlog_std::Vec<T>` though